live/index.html

```diff
@@ -1,583 +1,813 @@
-      <a onclick="glv.loadUrl('../data/klein-bottle.saved')">Klein Bottle</a>
-      <a onclick="glv.loadUrl('../data/laghos.saved')">Laghos</a>
-      <a onclick="glv.loadUrl('../data/mesh-explorer.saved')">Mesh Explorer</a>
-      <a onclick="glv.loadUrl('../data/mfem-logo.saved')">MFEM Logo</a>
-      <a onclick="glv.loadUrl('../data/minimal-surface.saved')"
-        >Minimal Surface</a
-      >
-      <a onclick="glv.loadUrl('../data/mobius-strip.saved')">Mobius Strip</a>
-      <a onclick="glv.loadUrl('../data/navier.saved')">Navier</a>
-      <a onclick="glv.loadUrl('../data/quad.saved')">Quad (default)</a>
-      <a onclick="glv.loadUrl('../data/remhos.saved')">Remhos</a>
-      <a onclick="glv.loadUrl('../data/shaper.saved')">Shaper</a>
-      <a onclick="glv.loadUrl('../data/snake.saved')">Snake</a>
-    </div>
-
-    <div id="log-side-nav" class="sidenav">
-      <a
-        href="javascript:void(0)"
-        class="closebtn"
-        onclick="closeNav('log-side-nav')"
-        >&times;</a
-      >
-
-      <div id="log-container">
-        <pre id="log"></pre>
-      </div>
-    </div>
-
-    <div id="side-menu" class="vertical">
-      <span onclick="openNav('about-side-nav')">About</span>
-      <span onclick="updateWindowSize()">Redraw</span>
-      <span onclick="openNav('log-side-nav', '43ch')">Log</span>
-      <span onclick="openNav('ctrl-side-nav')">Control</span>
-      <span onclick="openNav('setup-side-nav')">Setup</span>
-      <span onclick="openNav('help-side-nav', '40ch')">Help</span>
-      <span onclick="openNav('examples-side-nav')">Examples</span>
-    </div>
-
-    <script type="text/javascript">
-      var main_div = document.getElementById("main");
-      var side_menu = document.getElementById("side-menu");
-      var expand_btn = document.getElementById("expand-btn");
-
-      augmentLoggers("log", "log-side-nav");
-
-      function toggleSideMenu(show = true) {
-        if (show) {
-          side_menu.style.display = "block";
-          expand_btn.style.display = "block";
-        } else {
-          side_menu.style.display = "none";
-          expand_btn.style.display = "none";
-        }
-      }
+      this.glv.registerNewStreamCallback((state) => {
+        state.getHelpString().then((s) => {
+          this.helpString = s;
+        });
+      });
 
-      function pageToggleFullscreen() {
-        if (document.fullscreenEnabled) {
-          var elem = document.documentElement;
-          if (document.fullscreenElement === null) {
-            openFullscreen(elem);
-          } else {
-            closeFullscreen(elem);
-          }
-        } else {
-          console.warn("full screen not allowed");
-        }
+      var stream = new URLSearchParams(document.location.search).get("stream");
+      if (stream) {
+        console.log(`loading stream from ${stream}`);
+        this.glv.loadUrl(stream).then(() => {
+          this.glvIsInitializing = false;
+        });
+      } else {
```

Comment:
Do we still want to support the example stream, for when people are using this locally?

For example:

```
else if (example_stream !== undefined) {
  this.glv.displayStream(example_stream);
} else {
  this.glv.loadUrl(this.examples[this.defaultExample].url).then(() => {
    this.glvIsInitializing = false;
  });
}